chatbot/vector-db-persist-directory/resources/assignments.txt

```diff
@@ -0,0 +1,26 @@
+#### assignments:
```

Comment:
In program, we traverse the files included in this folder. And it is better to add some comments and add titles which is possible for developers to check or debug the data. This will not affect the progress of the program, because I have already dealt with this issue.

Reply:
Ok, I will delete the comments like "###"